### PR TITLE
Use node_exporter metrics for node CPU

### PR DESCRIPTION
**Pull request by SuperQ**
# Pull Request

### :mag_right: What kind of change is it?

- fix
 
### :dart: What has been changed and why do we need it?

- Use the node_exporter CPU metrics to get system level data.

### :heavy_check_mark: Which issue(s) this PR fixes?

- Fixes: https://github.com/dotdc/grafana-dashboards-kubernetes/issues/3

### :speech_balloon: Additional information?

- ...

### CODE REVIEW

**Comment by dotdc** [comment]
Thank you @SuperQ, will test this when I can :+1:

File: dashboards/k8s-views-global.json [hunk 1]
```diff
@@ -686,7 +686,7 @@
           },
           "editorMode": "code",
           "exemplar": true,
-          "expr": "sum(rate(container_cpu_usage_seconds_total{id=\"/\"}[$__rate_interval]))",
+          "expr": "sum(1-rate(node_cpu_seconds_total{mode=\"idle\"}}[$__rate_interval]))",
```
Comment (dotdc) [inline dashboards/k8s-views-global.json:689]: }}

Comment (SuperQ) [reply dashboards/k8s-views-global.json:689]: Oops, good catch

**Pull request merged**